shell/src/main/java/alluxio/extension/ExtensionUtils.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.extension;
+
+import alluxio.cli.ExtensionShell;
+import alluxio.cli.CommandUtils;
+import alluxio.cli.validation.Utils;
+import alluxio.extension.command.ExtensionCommand;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Class for convenience methods used by instances of {@link ExtensionShell}.
+ */
+@ThreadSafe
+public final class ExtensionUtils {
+  public static final String SSH_OPTS = "-o StrictHostKeyChecking=no -o ConnectTimeout=5";
```

Comment:
are these options specific to extensions, or should we use them for any ssh commands we invoke?